validator/node/node.go

```diff
@@ -246,21 +248,23 @@ func (s *ValidatorClient) initializeFromCLI(cliCtx *cli.Context) error {
 }
 
 func moveDb(cliCtx *cli.Context, accountsDir string) string {
-	dataDir := cliCtx.String(cmd.DataDirFlag.Name)
+	dataDir := cmd.DefaultDataDir()
```

Comment:
Why doesn't this read the flag?

Reply:
i think reading the flag is indeed the right thing to do. wanted to check the default location in case there is a db there.
fixing

Reply:
@prestonvanloon i am worried about the case were db already moved to accounts dir and now the user resets it to diff location. adding handling for that case as well